### ISSUE TITLE
NameError: name 'exit' is not defined

### ISSUE THREAD

**Issue by Hvzec** [ProgrammingHero1/fantastic_IDE#1]
You have to apply the function to sys
`from sys import exit`

**Comment by ashraf-minhaj** [comment]
Or you can use just this instead of exit `quit()`

**Comment by AatifS9506** [comment]
Open this YouTube link: https://www.youtube.com/watch?v=f1u3me4GYmw
You'll remember how to define 'exit'

**Comment by CoderKid12344** [comment]
> You have to apply the function to sys
> `from sys import exit`

no, exit is a by default function in python!! I didnt import sys
